lib/topological_inventory/satellite/operations/source.rb

```diff
@@ -37,6 +37,7 @@ def initialize(params = {}, request_context = nil, receptor_client = nil)
         def availability_check
           return if params_missing?
 
+          logger.info("Checking availability of source #{source_id}")
```

Comment:
This is duplicate to https://github.com/RedHatInsights/topological_inventory-satellite/blob/master/lib/topological_inventory/satellite/operations/processor.rb#L23 (actually it contains less info)

Reply:
Okay, given that we already have log messages at the start and end of every processed message, what, exactly is needed for https://projects.engineering.redhat.com/browse/TPINVTRY-839 ?

As I understand it, you were having trouble debugging issues with availability checks, right @slemrmartin @gtanzillo ?

Reply:
I thought logging of operations was implemented by @lpichler by adding request_id - to be able to track catalog orders among services (catalog -> topological api -> operations -> api -> catalog)

There is nothing like that for collector -> ingress api -> persister if we need that. 

But I didn't hear about any issue with availability check debugging, @gtanzillo?

Reply:
@carbonin maybe it was thought as extending existing log messages in sources-monitor -> sources api -> operations -> sources api to have some common pattern for filtering availability checks in kibana among services

Reply:
@carbonin @slemrmartin This is one of the supportability items that Adam and I identified as necessary for supporting Topo when it goes GA. When I was troubleshooting the Satellite availability issue this week I was not able to find any log message pertaining to checking the specific source that the fifi team pointed me to. If we have that loggin, perhaps you can point me to it. But it seems like we're still flying blind.

Reply:
@gtanzillo I'm able to track it now, but it's not as easy as it should be. So if I understand we need what I described in my previous comment

Reply:
So I would delete this message